README.md

```diff
@@ -26,20 +28,23 @@ The datamodel can be found at `./database/datamodel.graphql`. By the end of the
 
 Some goals could look like the following, but not limited to:
 
-- [ ] Update and Delete functionality
+- [ ] CRUD functionality for WineTasters
+- [ ] Include a small Login Component to start supporting Users in the app.
 - [ ] Review expanded with fields such as: predicted price, suggested pairing, or predicted year
-- [ ] WineTaster expanded to include other user profile fields
-- [ ] Wines have an attached image (a placeholder image is found in `./src/assets`)
-- [ ] TastingSessions have a date
+- [ ] Include another section in the app to search reviews and filter them based on different parameters (such as best/worst rated, date, price).
+- [ ] Support the ability to use enviroment variables. Both in the Front and Back of the app.
+
```

Comment:
Front end updates look good, but there was the bug we saw when you demonstrated the work to us yesterday. We could not see the taster being added I think is what it was. Could we include the bug - and instructions to fix it in the ReadMe?

Reply:
I remembered that the thing that supposed to be a bug wasn't really a bug. I just didn't worked on showing the list of WineTasters on that page. I can add that as a feature too. Thoughts?

Reply:
Sure - as longs as its well described